.env.example

```
@@ -42,3 +42,11 @@ S3_USE_PATH_STYLE=true
 
 # NATS
 NATS_PORT=4222
+
+# Queue behavior (Issue #9)
+QUEUE_STREAM_NAME=jobs_stream
+QUEUE_SUBJECT=jobs.execute
+QUEUE_DEAD_LETTER_SUBJECT=jobs.dead_letter
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The QUEUE_DEAD_LETTER_SUBJECT key should go before the QUEUE_STREAM_NAME key

(UnorderedKey)